src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,80 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.NoSuchElementException;
+
 public class WorkWithFile {
+    private static final String SPLITERATOR = ",";
+    private static final String CASE_BUY = "buy";
+    private static final String CASE_SUPPLY = "supply";
+    private static final String RESULT = "result";
+    private static final String EXCEPTION_MESSAGE = "Key doesn't found";
+    private static final int SPECIFIC_FIRST_ELEMENT = 0;
+    private static final int SPECIFIC_SECOND_ELEMENT = 1;
+    private static final String SPLIT_BY_NEW_LINE = "\n";
+    private static final int END_FILE_CODE = -1;
+    private static final int SET_LENGTH_TO_ZERO = 0;
+    private final StringBuilder textForSavingToFile = new StringBuilder();
+    private final StringBuilder constructTextFromFile = new StringBuilder();
```

Comment:
Don`t init fields

Reply:
Why don't we need to initialize fields  StringBuilder at class level?